src/components/Badge/Badge.st.css

```diff
@@ -0,0 +1,43 @@
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: Badge;
```

Comment:
I see you're importing the Badge style from `wix-ui-core` but you haven't used the component in the `*.tsx` file.
Is there a reason for this?

Reply:
Committed by mistake, do not need it